build.gradle

```diff
@@ -75,8 +81,8 @@ jooq {
             //driver = 'com.mysql.cj.jdbc.Driver'
             driver = 'com.mysql.jdbc.Driver'
             url = 'jdbc:mysql://localhost:3306/?useSSL=false&serverTimezone=UTC'//change to true for production
-            user = 'fcgl-user'//Change to your user
-            password = 'password'//Change to your password
+            user = 'user'//Change to your user
+            password = 'pass'//Change to your password
```

Comment:
We shouldn't be changing this. Let's agree on one standard and not change it anymore

Reply:
These should be in a property file, not a build file. But yea... TODO